src/v2/Apps/ViewingRoom/routes.tsx

```diff
@@ -43,7 +43,6 @@ export const routes: RouteConfig[] = [
       {
         path: "/",
         Component: StatementRoute,
-        ignoreScrollBehavior: true,
```

Comment:
I think this means we'll get that pop-to-top behavior if we navigate from works -> statement as well, but that feels better than the current. We can play around!

Reply:
Yes, because `/statement` is the same as `/` and is a default. But the hope is that people don't click between tabs too much:)